pkg/codegen/internal/test/testdata/simple-resource-schema/dotnet/Resource.cs

```diff
@@ -61,7 +65,23 @@ public static Resource Get(string name, Input<string> id, CustomResourceOptions?
     public sealed class ResourceArgs : Pulumi.ResourceArgs
     {
         [Input("bar")]
-        public Input<string>? Bar { get; set; }
+        public Input<string>? Bar
+        {
+            get => Bar;
```

Comment:
This isn't what you want -- you're just calling `Bar` recursively, which is why you'll run into a `StackOverflowException`. If you're going to customize the getter/setter implementation, you'll need to add an explicit private backing field, e.g.:

```csharp
private Input<string>? _bar;

[Input("bar")]
{
    get => _bar;
    set
    {
        if (value != null)
        {
            // Always mark this field as secret to avoid leaking sensitive values into the state.
            // Since we can't directly assign the Output from CreateSecret to the property, use an
            // Output.Tuple to enable the secret flag on the data.
            var emptySecret = Output.CreateSecret(0);
            _bar = Output.Tuple<Input<string>?, int>(value, emptySecret).Apply(t => t.Item1);
        }
        else
        {
            _bar = null;
        }
    }
}
```